torch/fx/experimental/rewriter.py

```diff
@@ -38,8 +38,11 @@ def rewrite(self, fn: FunctionType):
         keys_before = set(globals_dict.keys())
         exec(code, globals_dict)
         new_keys = list(set(globals_dict.keys()) - keys_before)
-        assert len(new_keys) == 1
-        fn_compiled = globals_dict[new_keys[0]]
+        assert len(new_keys) <= 1
+        if len(new_keys) == 0:
+            fn_compiled = globals_dict["forward"]
```

Comment:
I just encountered another case where it's not "forward" but "_forward_unimplemented". I'm wondering if it makes sense to make it like 
```
assert len(new_keys) <= 1
fn_compiled = globals_dict[fn.__name__]
```

Reply:
We always symbolically trace the function called `forward` (refer to [this](https://github.com/pytorch/pytorch/blob/master/torch/fx/symbolic_trace.py)), so I don't think it makes sense to allow for a non-forward function to be used with the subgraph rewriter. It looks like [_forward_unimplemented](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/module.py#L190) is an intermediate state for the `forward` function. How are you getting to the point where you're rewriting `_forward_unimplemented`?

Reply:
I see. I just found out that I was seeing "_forward_unimplemented` because I was using acc rewriter which is slightly different from this one and not related to this PR. Sorry for the confusion.